filtered-resource-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/filteredresource/redis/RedisPersistenceContractTest.java

```diff
@@ -50,40 +42,18 @@
 
 @DataJpaTest(properties = {"spring.data.redis.repositories.timeToLive.TokenOffsetEntity=1s"})
```

Comment:
As a proof-of-concept, can you add a keyPrefix in here like we do for ams/data-svc?

Reply:
Updated